corehq/ex-submodules/casexml/apps/case/dbaccessors/by_owner.py

```diff
@@ -14,8 +32,13 @@ def get_closed_case_ids(owner_id):
 
 def _get_case_ids(owner_id, is_closed):
```

Comment:
Makes sense, though didn't feel the need to document too much since this function is only internal the the module where the function names are pretty clear